README.md

```diff
@@ -1,2 +1,70 @@
+
+The first specification from the Notary Project is the [signature specification](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-specification.md) that specifies how portable signatures wrapped in [COSE](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-envelope-cose.md) or [JWS](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-envelope-jws.md) envelopes can be produced. The specification defines the [signing and verification workflow](https://github.com/notaryproject/notaryproject/blob/main/specs/signing-and-verification-workflow.md) (aka _Notary signing and verification_), the [signing scheme](https://github.com/notaryproject/notaryproject/blob/main/specs/signing-scheme.md), the signature format and how to wrap the signature using COSE or JWS envelopes. A signature, also called a _Notary signature_, produced according to the Notary signature specification can be copied between OCI registries and validated in connected, occasionally connected, and disconnected environments without the need of additional server insfrastructure. The Notary signature specification is available in the [notaryproject](https://github.com/notaryproject/notaryproject/tree/main/specs) repository under the Notary Project.
+
+The [notaryproject](https://github.com/notaryproject/notaryproject/) repository also contains information about the [requirements and scenarios](https://github.com/notaryproject/notaryproject/tree/main/requirements) that the Notary Project supports or plans to support as well as the [reports from security testing and audits](https://github.com/notaryproject/notaryproject/tree/main/security).
+
+A reference implementation for the Notary signature in Golang is provided in the [`notation-core-go`](https://github.com/notaryproject/notation-core-go) library. A convenient Golang library that interacts with OCI registries and manages the relation between a signed artifact and a signature is provided in the [`notation-go`](https://github.com/notaryproject/notation-go) library. `notation-go` provides an easy way to implement the Notary signing and verification in Golang. The convenient library is also used by the [`notation` CLI](https://github.com/notaryproject/notation) that can be used by developers and CI/CD pipelines to produce portable signatures and store them together with the signed artifacts in OCI-compliant registries. The `notation` CLI implements the Notary signing and verification and can also be used to verify signatures of artifacts stored in OCI-compliant registries.
```

Comment:
We should add that the libraries "notation-go" and "notation-core-go" can be used indepdently of the Notation CLI, and other projects like Ratify and take a direct dependency on those libraries without using Notation CLI.

Reply:
Added text to clarify thatl